website/content/v1.3/talos-guides/network/vip.md

```diff
@@ -5,30 +5,24 @@ aliases:
   - ../../guides/vip
 ---
 
-One of the biggest pain points when building a high-availability controlplane
+One of the pain points when building a high-availability controlplane
 is giving clients a single IP or URL at which they can reach any of the controlplane nodes.
-The most common approaches all require external resources:  reverse proxy, load
-balancer, BGP, and DNS.
-
-Using a "Virtual" IP address, on the other hand, provides high availability
-without external coordination or resources, so long as the controlplane members
-share a layer 2 network.
-In practical terms, this means that they are all connected via a switch, with no
-router in between them.
-
-The term "virtual" is misleading here.
-The IP address is real, and it is assigned to an interface.
-Instead, what actually happens is that the controlplane machines vie for
-control of the shared IP address using etcd elections.
-There can be only one owner of the IP address at any given time, but if that
+The most common approaches - reverse proxy, load
+balancer, BGP, and DNS - all require external resources, and add complexity in setting up Kubernetes.
+
+To simplify cluster creation, Talos Linux supports a "Virtual" IP (VIP) address to access the Kubernetes API server, providing high availability with no other resources required.
+
+What happens is that the controlplane machines vie for control of the shared IP address using etcd elections.
```

Comment:
vie?

Reply:
Vie sort of means "compete" or "try to win".

Reply:
> Vie sort of means "compete" or "try to win".

TIL :+1: